index.html

```diff
@@ -519,12 +519,14 @@ <h6>JWT Decoding</h6>
       </section>
     </section>
 
+    <section id="version-1.1">
+      <h2>Version 1.1</h2>
```

Comment:
added a section so we can distinguish between production rules for 1.1 and potential production rules for future versions